pkg/operator/controller/status/controller.go

```diff
@@ -385,14 +402,28 @@ func checkAllIngressesAvailable(ingresses []operatorv1.IngressController) bool {
 	return len(ingresses) != 0
 }
 
-// computeOperatorDegradedCondition computes the operator's current Degraded status state.
-func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController) configv1.ClusterOperatorStatusCondition {
+// computeIngressControllerDegradedCondition computes the operator's current Degraded status state.
```

Comment:
This comment is still wrong.

Reply:
Fixed the name of the function in the comment.